risc0/zkvm/Cargo.toml

```diff
@@ -136,7 +136,7 @@ dual = []
 # The zkVM exposes a getrandom implementation that panics by default. This will
 # expose a getrandom implementation that uses the `sys_random` ecall.
 getrandom = ["risc0-zkvm-platform/getrandom"]
-metal = []
+metal = ["prove"]
```

Comment:
```suggestion
metal = ["prove", "risc0-zkp/metal"]
```

Reply:
Also here, I was under the assumption that by including `prove` we get 
```
"risc0-circuit-recursion/prove",
 "risc0-circuit-rv32im/prove",
 "risc0-zkp/prove",
 ```
 Wouldn't that be enough?